<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>SaveGestureDialog</class>
  <widget class="QDialog" name="SaveGestureDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>951</width>
        <height>600</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>790</width>
        <height>600</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>1360</width>
        <height>600</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Zapis Å›ciezki</string>
    </property>
    <property name="modal">
      <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
        <widget class="QFrame" name="frame">
          <property name="maximumSize">
            <size>
              <width>16777215</width>
              <height>40</height>
            </size>
          </property>
          <layout class="QHBoxLayout" name="_2" stretch="0,0,0">
            <property name="spacing">
              <number>6</number>
            </property>
            <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
              <number>1</number>
            </property>
            <property name="topMargin">
              <number>1</number>
            </property>
            <property name="rightMargin">
              <number>1</number>
            </property>
            <property name="bottomMargin">
              <number>1</number>
            </property>
            <item>
              <spacer>
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>131</width>
                    <height>31</height>
                  </size>
                </property>
              </spacer>
            </item>
            <item>
              <widget class="QPushButton" name="okButton">
                <property name="text">
                  <string>Zapisz</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="cancelButton">
                <property name="text">
                  <string>Anuluj</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <widget class="QFrame" name="frame_3">
              <property name="maximumSize">
                <size>
                  <width>260</width>
                  <height>16777215</height>
                </size>
              </property>
              <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
                <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                  <widget class="QFrame" name="frame_4">
                    <property name="maximumSize">
                      <size>
                        <width>16777215</width>
                        <height>16777215</height>
                      </size>
                    </property>
                    <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QFormLayout" name="formLayout_2">
                      <item row="0" column="0">
                        <widget class="QLabel" name="label">
                          <property name="text">
                            <string>Nazwa</string>
                          </property>
                        </widget>
                      </item>
                      <item row="0" column="1">
                        <widget class="QTextEdit" name="inGestureName">
                          <property name="sizePolicy">
                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                              <horstretch>0</horstretch>
                              <verstretch>0</verstretch>
                            </sizepolicy>
                          </property>
                          <property name="minimumSize">
                            <size>
                              <width>0</width>
                              <height>0</height>
                            </size>
                          </property>
                          <property name="maximumSize">
                            <size>
                              <width>16777215</width>
                              <height>25</height>
                            </size>
                          </property>
                          <property name="acceptDrops">
                            <bool>true</bool>
                          </property>
                          <property name="inputMethodHints">
                            <set>Qt::ImhNone</set>
                          </property>
                          <property name="verticalScrollBarPolicy">
                            <enum>Qt::ScrollBarAlwaysOff</enum>
                          </property>
                          <property name="horizontalScrollBarPolicy">
                            <enum>Qt::ScrollBarAlwaysOff</enum>
                          </property>
                          <property name="acceptRichText">
                            <bool>false</bool>
                          </property>
                        </widget>
                      </item>
                      <item row="1" column="0">
                        <spacer name="verticalSpacer">
                          <property name="orientation">
                            <enum>Qt::Vertical</enum>
                          </property>
                          <property name="sizeType">
                            <enum>QSizePolicy::MinimumExpanding</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                            <size>
                              <width>20</width>
                              <height>30</height>
                            </size>
                          </property>
                        </spacer>
                      </item>
                    </layout>
                  </widget>
                </item>
                <item>
                  <widget class="QFrame" name="frame_2">
                    <property name="frameShape">
                      <enum>QFrame::Box</enum>
                    </property>
                    <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QGridLayout" name="gridLayout">
                      <item row="0" column="1">
                        <widget class="QCheckBox" name="checkHand2">
                          <property name="text">
                            <string>2</string>
                          </property>
                        </widget>
                      </item>
                      <item row="1" column="1">
                        <widget class="QCheckBox" name="checkHand5">
                          <property name="text">
                            <string>5</string>
                          </property>
                        </widget>
                      </item>
                      <item row="1" column="2">
                        <widget class="QCheckBox" name="checkHand6">
                          <property name="text">
                            <string>6</string>
                          </property>
                        </widget>
                      </item>
                      <item row="0" column="0">
                        <widget class="QCheckBox" name="checkHand1">
                          <property name="text">
                            <string>1</string>
                          </property>
                        </widget>
                      </item>
                      <item row="1" column="0">
                        <widget class="QCheckBox" name="checkHand4">
                          <property name="text">
                            <string>4</string>
                          </property>
                        </widget>
                      </item>
                      <item row="0" column="2">
                        <widget class="QCheckBox" name="checkHand3">
                          <property name="text">
                            <string>3</string>
                          </property>
                        </widget>
                      </item>
                      <item row="9" column="0">
                        <widget class="QCheckBox" name="checkHand7">
                          <property name="text">
                            <string>7</string>
                          </property>
                        </widget>
                      </item>
                      <item row="9" column="1">
                        <widget class="QCheckBox" name="checkHand8">
                          <property name="text">
                            <string>8</string>
                          </property>
                        </widget>
                      </item>
                      <item row="9" column="2">
                        <widget class="QCheckBox" name="checkHand9">
                          <property name="text">
                            <string>9</string>
                          </property>
                        </widget>
                      </item>
                    </layout>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
          <item>
            <widget class="QGroupBox" name="groupBox">
              <property name="title">
                <string>Gest</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
                <item row="2" column="2">
                  <widget class="QScrollArea" name="scrollArea_6">
                    <property name="widgetResizable">
                      <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_11">
                      <property name="geometry">
                        <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>201</width>
                          <height>159</height>
                        </rect>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_7">
                        <item>
                          <widget class="QLabel" name="handPreview9">
                            <property name="text">
                              <string/>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
                <item row="2" column="1">
                  <widget class="QScrollArea" name="scrollArea">
                    <property name="widgetResizable">
                      <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_2">
                      <property name="geometry">
                        <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>201</width>
                          <height>159</height>
                        </rect>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <item>
                          <widget class="QLabel" name="handPreview8">
                            <property name="text">
                              <string/>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
                <item row="0" column="2">
                  <widget class="QScrollArea" name="scrollArea_5">
                    <property name="widgetResizable">
                      <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_10">
                      <property name="geometry">
                        <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>201</width>
                          <height>159</height>
                        </rect>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_6">
                        <item>
                          <widget class="QLabel" name="handPreview3">
                            <property name="text">
                              <string/>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
                <item row="2" column="0">
                  <widget class="QScrollArea" name="scrollArea_7">
                    <property name="widgetResizable">
                      <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_12">
                      <property name="geometry">
                        <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>201</width>
                          <height>159</height>
                        </rect>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_9">
                        <item>
                          <widget class="QLabel" name="handPreview7">
                            <property name="text">
                              <string/>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
                <item row="0" column="1">
                  <widget class="QScrollArea" name="scrollArea_2">
                    <property name="widgetResizable">
                      <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_7">
                      <property name="geometry">
                        <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>201</width>
                          <height>159</height>
                        </rect>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                          <widget class="QLabel" name="handPreview2">
                            <property name="text">
                              <string/>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
                <item row="0" column="3">
                  <widget class="QLabel" name="handPreview1_4">
                    <property name="text">
                      <string/>
                    </property>
                    <property name="alignment">
                      <set>Qt::AlignCenter</set>
                    </property>
                  </widget>
                </item>
                <item row="1" column="0">
                  <widget class="QScrollArea" name="scrollArea_8">
                    <property name="widgetResizable">
                      <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_13">
                      <property name="geometry">
                        <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>201</width>
                          <height>158</height>
                        </rect>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_8">
                        <item>
                          <widget class="QLabel" name="handPreview4">
                            <property name="text">
                              <string/>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
                <item row="1" column="1">
                  <widget class="QScrollArea" name="scrollArea_4">
                    <property name="widgetResizable">
                      <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_9">
                      <property name="geometry">
                        <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>201</width>
                          <height>158</height>
                        </rect>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_5">
                        <item>
                          <widget class="QLabel" name="handPreview5">
                            <property name="text">
                              <string/>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
                <item row="0" column="0">
                  <widget class="QScrollArea" name="scrollArea_9">
                    <property name="widgetResizable">
                      <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_15">
                      <property name="geometry">
                        <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>201</width>
                          <height>159</height>
                        </rect>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_3">
                        <item row="0" column="0">
                          <widget class="QLabel" name="handPreview1">
                            <property name="text">
                              <string/>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
                <item row="1" column="2">
                  <widget class="QScrollArea" name="scrollArea_3">
                    <property name="widgetResizable">
                      <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_8">
                      <property name="geometry">
                        <rect>
                          <x>0</x>
                          <y>0</y>
                          <width>201</width>
                          <height>158</height>
                        </rect>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_4">
                        <item>
                          <widget class="QLabel" name="handPreview6">
                            <property name="text">
                              <string/>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
